mycroft/client/enclosure/enclosure.py

```diff
@@ -202,16 +202,91 @@ class Enclosure:
     """
 
     def __init__(self):
-        self.__init_serial()
+        try:
```

Comment:
This logic together with the platform detection method should use our enclosure reader and writer instead. I'll write the flow in the review comments.